mix.exs

```diff
@@ -1,27 +1,14 @@
-defmodule Mix.Tasks.Compile.Exfswatch do
```

Comment:
That makes sense.

How would you feel about moving it back into the mix.exs but keeping it in the code: i.e.

```elixir